dbms/src/main/java/org/frankframework/dbms/Db2DbmsSupport.java

```diff
@@ -81,9 +81,7 @@ public boolean isColumnPresent(Connection conn, String schemaName, String tableN
 	}
 
 	public boolean hasIndexOnColumn(Connection conn, String schemaName, String tableName, String columnName) throws DbmsException {
-		List<String> columns = new LinkedList<>();
-		columns.add(columnName);
-		return hasIndexOnColumns(conn, schemaName, tableName, columns);
+		return hasIndexOnColumns(conn, schemaName, tableName, Collections.singletonList(columnName));
```

Comment:
Same, deze is ook immutable

Reply:
En dat is hier geheel niet relevant, gezien hoe het wordt gebruikt in de method die we aanroepen.